docs/api.md

```diff
@@ -501,6 +507,140 @@ Failing requests may be due to the following errors:
 * status code 413, errno 113:  request body too large
 * status code 401, errno 115:  invalid authentication nonce
 * status code 400, errno 120:  incorrect email case
+* status code 400, errno 121:  account is locked
+
+## POST /v1/account/lock
+
+HAWK-authenticated.
+
+This locks an account and prevents the user from performing any action that requires a password until their email address is re-verified. This endpoint is for testing only and is disabled for production.
+
+__Parameters__
+
+* email - the email address for the account
+* authPW - the PBKDF2/HKDF stretched password as a hex string
+
+### Request
+
+```sh
+curl -v \
+-X POST \
+-H "Content-Type: application/json" \
+https://api-accounts.dev.lcip.org/v1/account/lock \
+-d '{
+  "email": "me@example.com",
+  "authPW": "996bc6b1aa63cd69856a2ec81cbf19d5c8a604713362df9ee15c2bf07128efab"
+}'
+```
+
+### Response
+
+Successful requests will produce a "200 OK" and a json body.
+
+```json
+{}
+```
+
+Failing requests may be due to the following errors:
+
+* status code 400, errno 102:  attempt to access an account that does not exist
+* status code 400, errno 103:  incorrect password
+* status code 400, errno 106:  request body was not valid json
+* status code 400, errno 107:  request body contains invalid parameters
+* status code 400, errno 108:  request body missing required parameters
+* status code 411, errno 112:  content-length header was not provided
+* status code 413, errno 113:  request body too large
+* status code 400, errno 120:  incorrect email case
```

Comment:
Huh, so I added a test for this, and lockAccount does not seem to care about the email case.

```diff
diff --git a/test/remote/account_unlock_tests.js b/test/remote/account_unlock_tests.js
index 4779e30..a616db6 100644
--- a/test/remote/account_unlock_tests.js
+++ b/test/remote/account_unlock_tests.js
@@ -184,6 +186,41 @@ TestServer.start(config)
   )
 
   test(
+    'lock an account with an invalid email case',
+    function (t) {
+      var email = server.uniqueEmail()
+      var password = 'allyourbasearebelongtous'
+      var client = null
+      var code = null
+      return Client.createAndVerify(config.publicUrl, email, password, server.mailbox)
+      .then(
+        function () {
+          return Client.login(config.publicUrl, email, password)
+        }
+      )
+      .then(
+        function (x) {
+          client = x
+        }
+      )
+      .then(
+        function () {
+          return client.lockAccount(email.toUpperCase(), password)
+        }
+      )
+      .then(
+        function () {
+          t.fail('lockAccount is expected to fail')
+        },
+        function (err) {
+          t.equal(err.code, 400, '400 status code')
+          t.equal(err.errno, 120, 'incorrect email case errorno')
+        }
+      )
+    }
+  )
+
+  test(
     'teardown',
     function (t) {
       server.stop()
```

What is the expected behavior here?